src/screens/ActivitiesPage.js

```diff
@@ -21,7 +22,7 @@ const ActivitiesPage = () => {
           description={'Understanding how our decisions can impact others and ourselves'} 
           progress={72}
           color={'#00A87D40'} 
-          internalLink={'/activities'} />
+          internalLink={'/categories/responsible-decision-making'} />
```

Comment:
update each link i.e. `/categories/relationship-skills` to `/relationship-skills`